sway-lsp/src/traverse/parsed_tree.rs

```diff
@@ -405,7 +405,9 @@ impl Parse for AmbiguousPathExpression {
         let AmbiguousPathExpression {
             call_path_binding,
             args,
+            qualified_path_root: _,
         } = self;
+        // TODO: process qualified paths
```

Comment:
Is this still to do?

Reply:
Removed the to-do in the most recent version and added processing of qualified path tokens.